scipy-notebook/Dockerfile

```diff
@@ -19,27 +19,27 @@ USER $NB_UID
 # use notebook-friendly backends in these images
 RUN conda install --quiet --yes \
     'blas=*=openblas' \
-    'ipywidgets=7.1*' \
-    'pandas=0.19*' \
+    'ipywidgets=7.2*' \
+    'pandas=0.22*' \
     'numexpr=2.6*' \
-    'matplotlib=2.0*' \
-    'scipy=0.19*' \
-    'seaborn=0.7*' \
+    'matplotlib=2.1*' \
+    'scipy=1.0*' \
```

Comment:
We just went to 1.1.0. Not sure if you want to be bleeding edge though.

ref: https://github.com/conda-forge/scipy-feedstock/pull/81